xtask/src/cli/devtools_test.rs

```diff
@@ -0,0 +1,12 @@
+use clap::Args;
+
+#[derive(Args, Debug)]
+pub(crate) struct DevtoolsTestArg {
+    #[arg(
+        long,
```

Comment:
Can we also have the long `--accept` format?

Reply:
Yes, right now it is: `  -a, --snapshot-accept  Accept newly generated macro-snapshots`, which I kind of liked... Do you want the long to just be accept?

Reply:
Should we rename it into `--accept-snapshot`? That way `-a` aligns with `--accept..`

Reply:
I like that `-a` is accept and `--snapshot-accept`.  🤷 can we have it this way?

Reply:
Done! Was waaay easier than I thought.